addon/type-tests/modifier-test.ts

```diff
@@ -264,17 +265,14 @@ class ClassBased extends Modifier<ClassBasedSignature> {
   }
 }
 
-const classBased = new ClassBased(
-  { iAmAnOwner: 'yep' },
-  {
-    named: {
-      onMessage(desc, data) {
-        console.log(desc, JSON.stringify(data, null, 2));
-      },
+const classBased = new ClassBased({ iAmAnOwner: 'yep' } as unknown as Owner, {
```

Comment:
(sorry for indentation, it was not me but prettier) as we don't have real owner here, this should be ok, I think

Reply:
Seems fine, yeah; that's an annoying thing for tests. We *should* also be able to use the owner from the test context if it's useful, but there's a known gap in the need to be able to get an owner otherwise.